arrow-core-data/src/main/kotlin/arrow/core/NonEmptyList.kt

```diff
@@ -206,7 +219,7 @@ class NonEmptyList<out A>(
   inline fun <B> map(f: (A) -> B): NonEmptyList<B> =
     NonEmptyList(f(head), tail.map(f))
 
-  inline fun <B> flatMap(f: (A) -> NonEmptyListOf<B>): NonEmptyList<B> =
+  fun <B> flatMap(f: (A) -> NonEmptyListOf<B>): NonEmptyList<B> =
```

Comment:
This should stay inlined so it can ingest `suspend` functions.

Reply:
The problem of keeping it inlined is the compiler throws an error that says the method is too long when compiling `mapN`. `flatMap` is called several times in that method so the code is copied several times too, causing the above-mentioned error.

Reply:
🤯 That's bad because we want `mapN` to be inlined too. I've never ran into such an error actually.

cc\\ @raulraja

Reply:
I've never ran into that error either, @franciscodr do you have an example where we can reproduce this?